custom_components/tahoma/entity.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, device_url: str, coordinator: TahomaDataUpdateCoordinator):
         """Initialize the device."""
         super().__init__(coordinator)
         self.device_url = device_url
+        self.base_device_url, *_ = self.device_url.split("#")
```

Comment:
Isn't this entry already present in the executor? self.executor.base_device_url?

Reply:
Yes indeed. But it sounds weird to retrieve the executor to get this data while device_url is directly accessible.